Spanner/src/TransactionalReadTrait.php

```diff
@@ -363,6 +384,11 @@ public function id()
         return $this->transactionId;
     }
 
+    public function &getIdReference()
```

Comment:
This is again not necessary, we do not want to use pass by reference anywhere in this library.

Also, we should name things consistently, so this function should be `getTransactionId`,